bluepyopt/electrical/objectives.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, name, feature):
 
         Objective.__init__(self, name, [feature])
 
+    @property
```

Comment:
Yeah, was just going to comment on that.

MaxObjective and WeightedSumObjective both wrap that method w/ '@property', so to keep the same interface, I think it should have it too